src/printer.js

```diff
@@ -628,15 +628,17 @@ function genericPrintNoParens(path, options, print) {
           concat([
             leftBrace,
             indent(
-              options.tabWidth * (parentIsUnionTypeAnnotation ? 2 : 1),
+              options.tabWidth + (parentIsUnionTypeAnnotation ? 3 : 0),
```

Comment:
@jlongster @yamafaktory

I think this line should be indented 2 spaces:

`options.tabWidth + (parentIsUnionTypeAnnotation ? 2 : 0)`

Because when indenting a normal object, the opening brace is not counted.

```javascript
{
  key: "value",
}
```

So, from the characters `| {` only the pipe and the space should be counted.